packages/zone/src/heap.js

```diff
@@ -38,13 +35,15 @@ const detachedHeapStores = Far('heapStores', {
  */
 export const makeHeapZone = (baseLabel = 'heapZone') => {
   const { makeOnce, wrapProvider } = makeOnceKit(baseLabel, detachedHeapStores);
+
+  const subZoneProvider = (label, _options) =>
+    makeHeapZone(`${baseLabel}.${label}`);
+
   return Far('heapZone', {
     exo: wrapProvider(makeExo, keys.exo),
     exoClass: wrapProvider(defineExoClass, keys.exoClass),
     exoClassKit: wrapProvider(defineExoClassKit, keys.exoClassKit),
-    subZone: (label, _options) => {
-      return makeOnce(label, () => makeHeapZone(`${baseLabel}.${label}`));
-    },
+    subZone: wrapProvider(subZoneProvider),
```

Comment:
```suggestion
    subZone: wrapProvider(provideSubZone),
```

Reply:
This renaming does not sound good to me because the argument function itself is not a `provide*` function according to our convention. The function returned by `wrapProvider` is a `provide*` function according to our convention.

Reply:
Then maybe `makeSubZone` instead?

Reply:
Done